### ISSUE TITLE
Meteor 1.7.0.4 - Error: Cannot find module '@babel/runtime/helpers/builtin/interopRequireDefault'

### ISSUE THREAD

**Issue by holmrenser** [meteor/meteor#10153]
After upgrading to Meteor 1.7.0.4 I get the following errors:

```
Error: Cannot find module '@babel/runtime/helpers/builtin/interopRequireDefault'
     at Function.Module._resolveFilename (module.js:547:15)
     at Function.resolve (internal/module.js:18:19)
     at Object.require (/.../.meteor/local/build/programs/server/boot.js:288:32)
     at makeInstallerOptions.fallback (packages/modules-runtime.js:604:18)
     at Module.require (packages/modules-runtime.js:230:14)
     at require (packages/modules-runtime.js:244:21)
     at livedata_connection.js (/.../.meteor/local/build/programs/server/packages/ddp-client.js:147:30)
     at fileEvaluate (packages/modules-runtime.js:322:7)
     at Module.require (packages/modules-runtime.js:224:14)
     at require (packages/modules-runtime.js:244:21)
```
```
Unable to resolve some modules:

  "@babel/runtime/helpers/builtin/interopRequireDefault" in /.../imports/ui/util/Dropdown.jsx (web.browser)
  "@babel/runtime/helpers/builtin/objectSpread" in /.../imports/ui/genetable/filteroptions/GenomeSelect.jsx (web.browser)
  "@babel/runtime/helpers/builtin/extends" in /.../imports/ui/util/Dropdown.jsx (web.browser)
  "@babel/runtime/helpers/builtin/objectWithoutProperties" in /.../imports/ui/util/Dropdown.jsx (web.browser)
                                              
If you notice problems related to these missing modules, consider running:
                                              
  meteor npm install --save @babel/runtime 
```

Running ```meteor npm install --save @babel/runtime``` does not fix the problem.

 I am using ```"@babel/runtime": "^7.0.0-rc.1"```

**Comment by holmrenser** [comment]
Running ```meteor npm install --save @babel/runtime@7.0.0-beta.55``` fixes this problem.

Given that @babel/runtime is now at rc1, does this mean meteor will update?

**Comment by benjamn** [comment]
No, not until Meteor 1.7.1, where we have already updated (try `meteor update —release 1.7.1-beta.33`).

You should have seen another warning when using Meteor 1.7.0.4 about pinning the older version of `@babel/runtime`. In fact, that warning was the primary reason we released 1.7.0.4. Glad you figured it out, anyway!

Babel may be nearing stability at last, but updating to the latest RC without all the corresponding adjustments that we’ve made in Meteor 1.7.1 would cause several breaking changes for Meteor 1.7.0.x apps.

Stability only means it’s supposed to keep working after you get through all the tricky work of updating.

**Comment by sumruti** [comment]
./node_modules/reactstrap/es/Input.js
Module not found: Can't resolve '@babel/runtime/helpers/esm/assertThisInitialized'
